NEWS.md

```diff
@@ -9,6 +9,8 @@ shiny 1.6.0.9000
 
 ### New features and improvements
 
+* All uses of `list(...)` have been replaced with `rlang::list2(...)`. By effect, `{rlang}`'s `!!!` operator may now be used to "splice" a list of argument values into `...`. We think this'll be particularly useful for passing a list of `tabPanel()` to their consumers (i.e., `tabsetPanel()`, `navbarPage()`, etc). For example, `tabs <- list(tabPanel("A", "a"), tabPanel("B", "b")); navbarPage(!!!tabs)`. (#3315 and #3328)
```

Comment:
```suggestion
* All uses of `list(...)` have been replaced with `rlang::list2(...)`. This means that you can use trailing `,` with out warning and use rlang's `!!!` operator to splice" a list of argument values into `...`. We think this'll be particularly useful for passing a list of `tabPanel()` to their consumers (i.e., `tabsetPanel()`, `navbarPage()`, etc). For example, `tabs <- list(tabPanel("A", "a"), tabPanel("B", "b")); navbarPage(!!!tabs)`. (#3315 and #3328)
```

Is that true about splicing though? I thought `tagList()` already did auto-splicing?

Reply:
Yea, I believe all of htmltools uses `dots_list()` now. I couldn't find any mention of it in the NEWS though so I figured it wouldn't hurt to say it just generally works now with any `...`?

Reply:
Also, shouldn't the suggestion be `without error` instead of `with out warning`?

Reply:
Ooops yeah. But I meant doesn't splicing already happen without `!!!`?

Reply:
Ohh, technically no, but in practice the difference usually important

```r
> str(tagList(list(tags$a(), tags$div())))
List of 1
 $ :List of 2
  ..$ :List of 3
  .. ..$ name    : chr "a"
  .. ..$ attribs : Named list()
  .. ..$ children: list()
  .. ..- attr(*, "class")= chr "shiny.tag"
  ..$ :List of 3
  .. ..$ name    : chr "div"
  .. ..$ attribs : Named list()
  .. ..$ children: list()
  .. ..- attr(*, "class")= chr "shiny.tag"
 - attr(*, "class")= chr [1:2] "shiny.tag.list" "list"
> str(tagList(!!!list(tags$a(), tags$div())))
List of 2
 $ :List of 3
  ..$ name    : chr "a"
  ..$ attribs : Named list()
  ..$ children: list()
  ..- attr(*, "class")= chr "shiny.tag"
 $ :List of 3
  ..$ name    : chr "div"
  ..$ attribs : Named list()
  ..$ children: list()
  ..- attr(*, "class")= chr "shiny.tag"
 - attr(*, "class")= chr [1:2] "shiny.tag.list" "list"
```